src/progress/web/method/proto.i

```diff
@@ -0,0 +1,162 @@
+/*********************************************************************
+* Copyright (C) 2005,2009 by Progress Software Corporation. All rights    *
+* reserved.  Prior versions of this work may contain portions        *
+* contributed by participants of Possenet.                           *
+*                                                                    *
+*********************************************************************/
+/*--------------------------------------------------------------------------
+    Library     : proto.i
+    Purpose     : Public function prototypes for files in src/web/method
+
+    Syntax      : {src/web/method/proto.i}
+    Author(s)   : B.Burton 
+    Created     : 02/03/97
+    Notes       : This file is included by cgidefs.i.  
+  ------------------------------------------------------------------------*/
+
+/* Make sure this file is only included once */
+&IF DEFINED(PROTO_I) = 0 &THEN
+&GLOBAL-DEFINE PROTO_I TRUE
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) From src/progress/pct/pctWSComp.p - Preprocessor variable proto_i should be SCOPED-DEFINED [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.GlobalDefine)